src/actions/mod.rs

```diff
@@ -447,16 +451,27 @@ impl ActionHandler {
                 return;
             }
         };
-        let config = self.fmt_config.lock().unwrap();
+
+        let mut config = self.fmt_config.lock().unwrap().get_rustfmt_config().clone();
+        let file_lines = format!(r#"[{{
+            "file": "{}",
+            "range": [{}, {}]
+        }}]"#, path.to_str().unwrap(), range.start.line + 1, range.end.line + 1);
```

Comment:
If this range is from rls-span, then you should use the `to_one_indexed`/`to_zero_indexed` API rather than manually adding 1. If it is not, then you should convert the range to an rls-span range in server.rs